process/block/preprocess/transactions.go

```diff
@@ -719,11 +724,14 @@ func (txs *transactions) createAndProcessScheduledMiniBlocksFromMeAsValidator(
 	mapSCTxs map[string]struct{},
 	randomness []byte,
 ) (block.MiniBlockSlice, error) {
-
 	if !txs.enableEpochsHandler.IsFlagEnabled(common.ScheduledMiniBlocksFlag) {
 		return make(block.MiniBlockSlice, 0), nil
 	}
 
+	defer func() {
```

Comment:
Not only for regular processing, but also for scheduled processing (both as leader and as validator).